S12-enums/basic.t

```diff
@@ -252,4 +252,12 @@ is-deeply do { my enum Foos (a => <42>); a.Str }, 'a',
     'Pair elements in the list given to enum declaration work';
 }
 
+{ # RT #129142
+    my enum FF <zero one two three>;
+    my enum GG <fee fie foo fum>;
+    eval-lives-ok "my enum FF <zero one two three>; my enum GG <fee fie foo fum>; { FF(GG(2)).perl }",
+                  'Coercing an enum from a coercion of an enum from an int works';
+    is-deeply FF(GG(2)).perl, "FF", "Coercing an enum from a coercion of an enum returns expected result";
```

Comment:
This line still has the same issue (will fail during the compilation of the test file)